teams/spec-contributors.toml

```diff
@@ -3,10 +3,10 @@ subteam-of = "spec"
 
 [people]
 leads = []
-members = [
+members = []
+alumni = [
     "chorman0773",
 ]
-alumni = []
 
```

Comment:
you should probably change the title, since this is not m-ou-se

Reply:
I just appended this to the original PR that was `m-ou-se`, but ok, I will.